tests/WP_SQLite_Translator_Tests.php

```diff
@@ -1027,6 +1027,31 @@ public function testAlterTableAddNotNullVarcharColumn() {
 		);
 	}
 
+	public function testAlterTableWithColumnFirstAndAfter() {
+		$this->assertQuery(
+			"CREATE TABLE _tmp_table (
+				id int(11) NOT NULL,
+				name varchar(20) NOT NULL default ''
+			);"
+		);
+
+		$this->assertQuery(
+			"ALTER TABLE _tmp_table ADD COLUMN new_first_column VARCHAR(255) NOT NULL DEFAULT '' FIRST"
+		);
```

Comment:
It would be good to also make assertions about the resulting table structure like this:
https://github.com/WordPress/sqlite-database-integration/blob/dea3efd7adaa1eb5fdc66248669a8f3707450c20/tests/WP_SQLite_Translator_Tests.php#L885-L907

Reply:
👍 Added in https://github.com/WordPress/sqlite-database-integration/pull/147/commits/c4efdfd59d6c362237bc1b08477182c4db7a44b5.

Reply:
👍 Let's add them after each `assertQuery()` here to check each is doing what is expected.

Reply:
OK! Added in https://github.com/WordPress/sqlite-database-integration/pull/147/commits/4615adc48f81d0e5fabfe918c500ad6de9bddead.